gammapy/makers/reduce.py

```diff
@@ -58,12 +58,23 @@ def __init__(
                 self._apply_cutout = False
             else:
                 self.cutout_width = 2 * self.offset_max
-        self.n_jobs = n_jobs
+        self._n_jobs = n_jobs
```

Comment:
Maybe rely on the `setter` rather than set the internal variable. This way the setter will be tested.